.github/workflows/pr-go-e2e-all.yml

```diff
@@ -0,0 +1,16 @@
+name: CLI E2E tests
```

Comment:
does the name need to be unique?

Reply:
Actually it needs to be the same as the filtered